src/methods/wallet-send-calls.ts

```diff
@@ -0,0 +1,80 @@
+import { rpcErrors } from '@metamask/rpc-errors';
+import type { Infer } from '@metamask/superstruct';
+import {
+  nonempty,
+  type,
+  string,
+  array,
+  object,
+  optional,
+  tuple,
+} from '@metamask/superstruct';
+import type {
+  Json,
+  JsonRpcRequest,
+  PendingJsonRpcResponse,
+} from '@metamask/utils';
+import { HexChecksumAddressStruct, StrictHexStruct } from '@metamask/utils';
+
+import {
+  validateAndNormalizeKeyholder,
+  validateParams,
+} from '../utils/validation';
+
+const SendCallsStruct = tuple([
+  object({
+    version: optional(nonempty(string())),
```

Comment:
this doesn't seem to be optional in the spec

Reply:
It's not technically required yet as we could assume no version is equivalent to `1.0`.

But I've made it required to be explicit and compliant with the spec.